### ISSUE TITLE
Add option for keep and display logs

### ISSUE THREAD

**Issue by SergiuToporjinschi** [SergiuToporjinschi/node-red-contrib-heater-controller#8]
Optionally the user should be able to see the a log when on and off was emitted

**Comment by zoltantakacshu** [comment]
Hi Sergiu,

I see in the latest release the changes are logged and even shown in the output. Additionally a new icon appeared in the UI "logs" - what I couldn't figure it out how is it working ...
How the UI part supposed to be working ?
What is the number of changes kept in the logs ? 
Wouldn't be better if the logs would be sent in a different node output than the standard one ? (Just an idea - still trying come up with the best use of this - as previously I was logging the changes and the heat states into a chart to allow me to review the history of the heating)
Anyway - thanks ! Great work, I really love the controller.

**Comment by SergiuToporjinschi** [comment]
@zoltantakacshu can you see the button? this is not finished yet :) that button supposed to be not visible yet :). I wanted to create a dialog to display the logs but unfortunately is not possible without hacks so I'm planing to output as node message.

**Comment by zoltantakacshu** [comment]
Yes, the button is visible if there are logs collected... And the logs are already in the the node output as well...
:)

**Comment by SergiuToporjinschi** [comment]
yes I've tested now and I can see that, but I think I should output this as different message and not attached to the standard output

**Comment by zoltantakacshu** [comment]
Great. the separate node output will help to build a chart based on the measured and set temperature.
Thanks!
